src/config/composer.ts

```diff
@@ -0,0 +1,20 @@
+export interface ComposerRequirements {
+    [key: string]: string
+}
+
+/**
+ * @title Package
```

Comment:
Not needed?

Reply:
It was when I created the `composer.schema.json` from this file. But I dropped that idea but did not deleted this annotation. So yes, I will delete it. Thanks!